src/contribution-guide/contributing.md

```diff
@@ -0,0 +1,24 @@
+# Contribute to Kubewarden docs
+
+_Kubewarden welcomes suggestions for improvement from all contributors, new and experienced!_
+
+You can contribute to Kubewarden documentation by
+
+- Improving existing content
+- Creating new content
+- (_Upcoming_) Helping with translation of the documentation
+
+## Getting Started
+
+Everybody is welcome to suggest changes by submitting a pull request
+or report a bug/typo with the help of a GitHub issue to the
```

Comment:
```suggestion
or reporting a bug/typo with the help of a GitHub issue to the
```

Reply:
Keeping the verb tense consistent (submitting, reporting). You might even say "by reporting" as well.